crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -114,7 +114,7 @@ async fn follower_process<A, Handler: DigestHandler<A> + Clone>(
             target_num_tasks = usize::min(committee.voting_rights.len() - 1, degree);
             peer_names = peer_names
                 .into_iter()
-                .filter(|name| committee.voting_rights.contains_key(name))
+                .filter(|name| committee.authority_exists(name))
```

Comment:
It would probably help to make `voting_rights` private, so that we don't get folks reverting this in the future.